projects/go-lib/src/lib/components/go-file-upload/go-file-upload.component.html

```diff
@@ -0,0 +1,53 @@
+<label class="go-form__label"
+       [attr.for]="id"
+       [ngClass]="{'go-form__label--dark': theme === 'dark'}">
+  {{ label }}
+</label>
+<div
+  goDragonDrop
+  class="go-file-upload go-file-upload__container"
+  [ngClass]="{ 'go-file-upload--disabled': state === 'selected' || isLoading }"
+  (click)="filePicker.click()"
+  (fileDropped)="onFilePicked($event)"
+>
+  <div class="go-file-upload__container" *ngIf="state === 'selecting' && !isLoading">
+    <go-icon icon="publish" iconModifier="large"></go-icon>
+    Drag and Drop File or Click to Browse
+  </div>
+  <div class="go-file-upload__container go-file-upload__container--loading" *ngIf="isLoading">
+    <go-loader *ngIf="isLoading"></go-loader>
```

Comment:
Can we change this to `loaderSize="small"`, I think it fits the space a little better.